content/tutorials/history.textile

```diff
@@ -26,10 +28,14 @@ languages:
   - php
   - ruby
   - swift
+
 level: easy
 reading_time: 10
+tags:
+    - History
```

Comment:
This is my bad as I didn't mention so in the sheet but can we please replace all occurrences of the tag 'History' with 'Message History'?

Reply:
Done (i only found one)